src/components/10-atoms/input-text/index.js

```diff
@@ -233,6 +241,11 @@ class AXAInputText extends AXAPopupMixin(NoShadowDOM) {
 
     this.isPlaceholderInCounter = this.counter && /##.*##/.test(this.counter);
     this.modelCounter = this.getCounterText;
+
+    // Firefox ignores the html autofocus attribute.
```

Comment:
What is the evidence for this? 

https://caniuse.com/#feat=autofocus says it works.
https://bugzilla.mozilla.org/show_bug.cgi?id=712130 is closed since 2 years.

We need to know whether Firefox has a fundamental problem or whether how we try to apply autofocus in the context of lit-element is the issue.